src/http-servers/json-server.js

```diff
@@ -0,0 +1,23 @@
+import http from 'http';
+const { log } = console;
+
+const port = process.env.port || 3000;
+//create a server object:
+http.createServer((req, res) => {
+   
+    res.writeHead(200, {'content-type': 'application/json'});
+
+    const product = {
+        id: 1,
+        name: 'Supreme T-Shirt',
+        brand: 'Supreme',
+        price: 99.99,
+        options: [
+            { color: 'blue' },
+            { size: 'XL' }
+        ]
+    };
+    res.write(JSON.stringify(product));
```

Comment:
you can simply do `res.end(JSON.stringify(product))` as you're not executing any further actions after this writing besides end

Reply:
Thanks, updated code.